java-lotto/src/main/java/domain/Lotto/LottoNo.java

```diff
@@ -0,0 +1,43 @@
+package domain.Lotto;
+
+import java.util.Objects;
+
+public class LottoNo implements Comparable<LottoNo> {
+    private final int number;
+    private static final int MIN = 1;
+    private static final int MAX = 45;
+
```

Comment:
```suggestion
    private static final int MIN = 1;
    private static final int MAX = 45;

    private final int number;
```

상수를 먼저 배치하고 멤버변수를 아래에 배치하는 것은 어떤가요?
큰 상관은 없는데 변수 선언 시 일관성있게 작성할 수 있습니다